frontend/packages/console-dynamic-plugin-sdk/src/api/useResolvedExtensions.ts

```diff
@@ -48,38 +40,30 @@ export const useResolvedExtensions = <E extends Extension>(
 
   const [resolvedExtensions, setResolvedExtensions] = React.useState<ResolvedExtension<E>[]>([]);
   const [resolved, setResolved] = React.useState<boolean>(false);
-  const [error, setError] = React.useState<any>(undefined);
+  const [errors, setErrors] = React.useState<any>(undefined);
 
   React.useEffect(() => {
     let disposed = false;
 
-    Promise.all(
+    // eslint-disable-next-line promise/catch-or-return
+    Promise.allSettled(
       extensions.map(async (e) => {
-        const resolvedProperties = await resolveCodeRefProperties(e);
-        return deepMergeExtensionProperties(e, resolvedProperties) as ResolvedExtension<E>;
+        const resolvedExtension = await resolveExtension<typeof e, any, ResolvedExtension<E>>(e);
+        return resolvedExtension;
```

Comment:
@christianvogt The reason for declaring above constant is to comply with ESLint [`no-return-await`](https://eslint.org/docs/rules/no-return-await) rule.

Following alternative is possible, I'm not sure which one is better:
```ts
// eslint-disable-next-line no-return-await
return await resolveExtension<typeof e, any, ResolvedExtension<E>>(e);
```

Reply:
@vojtechszocs I think it is kind of a mute point to use `async` `await` here instead of simply returning the promise result from `resolveExtension`.

```
extensions.map((e) => resolveExtension<typeof e, any, ResolvedExtension<E>>(e));
```

Reply:
Agreed, will update the code.